pytorch/src/main/java/org/bytedeco/pytorch/presets/torch.java

```diff
@@ -1735,7 +1735,8 @@
         ),
         @Platform(
             value = {"linux", "macosx", "windows"},
-            link = {"c10", "c10_cuda", "torch_cpu", "torch_cuda", "torch"},
+            linkpath = {"/usr/local/cuda/lib64/", "C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v11.8/lib/x64/"},
```

Comment:
For consistency, we should add the same paths as the presets for CUDA here: https://github.com/bytedeco/javacpp-presets/blob/master/cuda/src/main/java/org/bytedeco/cuda/presets/cudart.java#L46

Reply:
With a default installation on Linux though, it looks like that particular library is found in "/usr/local/cuda-11.8/extras/CUPTI/lib64/" on Linux and in "C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v11.8//extras/CUPTI/lib64/" on Windows, no?